homeassistant/components/rki_covid/config_flow.py

```diff
@@ -0,0 +1,62 @@
+"""Make the component configurable via the UI."""
+
+import logging
+from typing import Any, Dict, Optional
+
+from rki_covid_parser.parser import RkiCovidParser
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from . import get_coordinator
+from .const import ATTR_COUNTY, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class RKICovidNumbersConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """RKI Covid numbers config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    _options: Optional[Dict[str, Any]] = None
```

Comment:
```suggestion
    _options: dict[str, Any] | None = None
```
use `from __future__ import annotations`

Reply:
And don't seem to be needed

Reply:
> Optional[X] is equivalent to X | None
https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
I know but it's more clearer to write as such which is asked

Reply:
I'll keep this for now. The core is still using python 3.9

Reply:
I think that core preference is to use the option suggested by @gjohansson-ST, while importing `from __future__ import annotations`.

Reply:
adding the annotations import solved the warning. Thanks